TimesReactIOS.podspec

```diff
@@ -8,14 +8,20 @@ yoga_version = "0.54.2"
 
 Pod::Spec.new do |s|
   s.name         = "TimesReactIOS"
-  s.version      = "0.0.2"
+  s.version      = "0.0.1"
```

Comment:
Might be a good idea to pull this out from packages/ios-app/package.json and keep it in sync with the ios-app package version. Same for the git tag a couple lines below